### ISSUE TITLE
output the itms:// link as an output variable 

### ISSUE THREAD

**Issue by Rich86man** [bitrise-io/steps-bitrise-ios-deploy#5]
It would be really nice to get the direct download link for the OTA build. I'm running an enterprise app and I'd prefer to share the direct link rather than have users go to bitrise to download the app.

**Comment by viktorbenei** [comment]
Good idea, we might add this in a future version; just wanted to note that you can already do this with the https://github.com/bitrise-io/steps-amazon-s3-deploy step, which generates the required Plist file and the ITMS link as well.
